tests/state_test_util.go

```diff
@@ -122,10 +146,11 @@ func (t *StateTest) Subtests() []StateSubtest {
 
 // Run executes a specific subtest.
 func (t *StateTest) Run(subtest StateSubtest, vmconfig vm.Config) (*state.StateDB, error) {
-	config, ok := Forks[subtest.Fork]
-	if !ok {
+	config, eips, err := getVMConfig(subtest.Fork)
+	if err != nil {
```

Comment:
Maybe it could be modified to:

    if config, eips, err := getVMConfig(subtest.Fork); err != nil {